Clinical-Genomics/docker-base-images
# Simplify d4tools image

### This PR adds | fixes:
- Fix #9 

### How to test:
- Build the image
- Run a container and make sure d4tools is executed

### Review:

This [version](https://semver.org/) is a:
